### PR TITLE
[documentation]: Restructure docs and add docs for new features

### PR DESCRIPTION
**Pull request by erbesharat**
**What this PR does / why we need it**:
Restructure docs and add docs for new features.

Undocumented:
- [x] Path chaining
- [x] Predefined Regex
- [x] txtd.io Usage Guide
- [x] Regex placeholders
...

**Special notes for your reviewer**:
Still WIP but a quick review can help. Let me know what I've missed and will add them to the list.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by erbesharat** [comment]
Forgot to add the named regex placeholders in b25b517, will add them in another commit.

File: content/docs/config/_index/content.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  txtdirect {
+    enable host
+    prometheus {
+      address 192.168.0.1
+    }
+  }
+}
+```
+
+These features are documented in separate sections that you can find using the menu on left side of the page.
```
Comment (stp-ip) [inline content/docs/config/_index/content.md]: Binding content to the layout is problematic. Maybe we can reword this to work without taking into account the layout?

File: content/docs/config/_index/content.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## CLI Flags
+
+Use the `-conf` flag to pass the config file to TXTDirect like this example:
```
Comment (stp-ip) [inline content/docs/config/_index/content.md]:
```suggestion
Use the `-conf` flag to pass a config file to TXTDirect.
```

File: content/docs/config/basics/types.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
++++
```
Comment (stp-ip) [inline content/docs/config/basics/types.md:1]: Not sure I like the design of using a table. Will think about a different way.

File: content/docs/config/basics/content.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+weight = 200
+
+title = "Basics"
+
+[sidebar]
+  enable = true
+  sticky = true
++++
+
+# Enable types
```
Comment (stp-ip) [inline content/docs/config/basics/content.md]: Configuring types could be the section heading and then no subheading. Too much noise I think.

Each example is quite small so just having the description + example with a bit of whitespace should be enough.

File: content/docs/config/basics/content.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+}
+```
+
+# DNS Resolver
+
+---
+
+To use a custom DNS resolver instead of the machine's default resolver, you can provide the `resolver` address in the config.
+
+## Config
```
Comment (stp-ip) [inline content/docs/config/basics/content.md]: Probably not necessary. Removing the heading makes the example easier to see and reduces the noise.

File: content/docs/config/_global/nav.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
++++
```
Comment (stp-ip) [inline content/docs/config/_global/nav.md]: Why the need for a new/overwritting nav?

Comment (stp-ip) [reply content/docs/config/_global/nav.md]: Should be inherited from the parent `/docs/`section I think.

File: content/docs/placeholders/placeholders.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
++++
```
Comment (stp-ip) [inline content/docs/placeholders/placeholders.md:1]: Should we add the regex placeholders as well with a link to their usage?

File: content/docs/placeholders/content.md [hunk 8]
```diff
@@ -8,25 +8,47 @@ title = "Placeholders"
...
+When a request goes through a `path` record, TXTDirect will add the regex
+matches after parsing the path to the placeholders so you can use them inside
+your records to have more dynamic redirects. For example, after parsing the
+`/test/path`, you can use the `{1}` and `{2}` placeholders to access the request
+path as placeholders. Each number represents a part of the parsed path and the
+order depends on the Regex that parsed the request path.
+
+### Named Regex Placeholders
+
+Just like how the normal regex placeholders work, you can also use the regex
```
Comment (stp-ip) [inline content/docs/placeholders/content.md]: `Just` is not helpful for beginners similar to saying "easily configure". It might not be easy for first time users and discouraging with language is problematic.

File: content/docs/placeholders/content.md [hunk 9]
```diff
@@ -8,25 +8,47 @@ title = "Placeholders"
...
+the `>` symbol. Like `{>Authorization}` or `{>Server}`.
+
+## Regex Placeholders
+
+When a request goes through a `path` record, TXTDirect will add the regex
+matches after parsing the path to the placeholders so you can use them inside
+your records to have more dynamic redirects. For example, after parsing the
+`/test/path`, you can use the `{1}` and `{2}` placeholders to access the request
+path as placeholders. Each number represents a part of the parsed path and the
+order depends on the Regex that parsed the request path.
```
Comment (stp-ip) [inline content/docs/placeholders/content.md]: Not sure, but either we mentioned the default regex or add an example of a regex. It seems that without that context I wouldn't be able to understand where the specific placeholders are coming from. Probably go with default regex. Aka each path segment is used as a numbered placeholder and then link to the specific docs on regexing.

File: content/docs/placeholders/placeholders.md (hunk 7 above)
Comment (stp-ip) [reply content/docs/placeholders/placeholders.md:1]: You are explaining them in the dynamic section, but as an overview it might be nice to have {N} as well as {<namedRegex>} or so.

Sidenote: What do we do, when a named regex collides with another placeholder? Should we prefix numbered and/or named regex placeholders with something such as {_1}/{_my-named-regex} or {RE_1}/{RE_my-named-regex} or {re_1}/{re_my-named-regex} etc.?

File: content/docs/config/_index/content.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+title = "Configuration"
+weight = 100
+
+[sidebar]
+  enable = true
++++
+
+TXTDirect uses [Caddy's](https://caddyserver.com) config controller so all the config files should follow the [Caddyfile syntax](https://caddyserver.com/v1/tutorial/caddyfile) rules.
+
+TXTDirect's reads its config from the `txtdirect` scope in the Caddyfile. So for example, using the following config you can tell TXTDirect to enable the `host` redirect type on `example.com` host and write the logs to `stdout`:
```
Comment (stp-ip) [inline content/docs/config/_index/content.md]:
```suggestion
TXTDirect's reads its config from the `txtdirect` scope in the Caddyfile. For example, using the following config you can tell TXTDirect to enable the `host` redirect type on the `example.com` host and write the logs to `stdout`:
```

File: content/docs/config/basics/content.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+You can either use a file path or outputs like `stdout` and `stderr` as TXTDirect's logs output.
+To configure a custom log output you can use the `logfile` field in the config:
+
+```
+txtdirect {
+  logfile /path/to/file
+}
+```
+
+Supported values are `stdout`, `stderr` or path to a file. If empty, logs will get
```
Comment (stp-ip) [inline content/docs/config/basics/content.md]:
```suggestion
Supported values are `stdout`, `stderr` or a file path. If empty, logs will get
```

File: content/docs/config/basics/types.md (hunk 3 above)
Comment (stp-ip) [reply content/docs/config/basics/types.md:1]: Or maybe we have to improve the table design. Would be ok with merging, but opening a separate issue for iterating on the tables (one issue for both redirect types as well as placeholders)

File: content/docs/config/fallback/content.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+[sidebar]
+  enable = true
+  sticky = true
++++
+
+## Flow
+
+---
+
+Step-by-Step fallback flow:
```
Comment (stp-ip) [inline content/docs/config/fallback/content.md:16]: Great documentation.

One thing that struck be was the difference in preference.
For the current TXT we do to > website > root, but for the parent we use website > root > to. Does that make sense? Any thoughts?

File: content/docs/hosted/_index/subdomains.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
++++
```
Comment (stp-ip) [inline content/docs/hosted/_index/subdomains.md:1]: Great overview.

File: content/docs/hosted/_index/content.md [hunk 14]
```diff
@@ -0,0 +1,18 @@
...
++++
+
+To easily try out or let us manage the infrastructure, we provide a hosted instance of TXTDirect with global anycasted IPs and a distributed set of servers to redirect your users as fast as possible.
+
+Take a look at our [hosted version](/hosted/) or jump right into the
+[Usage Guide](/docs/hosted/#basics).
+
+- [Basics](#basics)
+- [Subdomains](#subdomains-header)
+- [Subdomains Table](#subdomains)
```
Comment (stp-ip) [inline content/docs/hosted/_index/content.md:18]: Seems like a bit of duplication and not sure, if we should merge both guides later on. Still think a step by step similar to yc.txtdirect.com might come in handy.

File: content/docs/hosted/_index/basics.md [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+
+```
+a.example.com                    86000 IN CNAME   txtd.io.
+```
+
+Note: Keep in mind that `CNAME` records only work on subdomains, so if you want
+to redirect the incoming requests to your root record, you have to add `A` and
+`AAAA` records to your zone file.
+
+The records for root redirect should look like this:
```
Comment (stp-ip) [inline content/docs/hosted/_index/basics.md]:
```suggestion
The records for a root redirect should look like this:
```

File: content/docs/hosted/_index/basics.md [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+git.example.com             86000 IN CNAME   txtd.io.
+_redirect.git.example.com   86000 IN TXT     "v=txtv0;type=host;to=https://github.com/example"
+```
+
+So the example above points `git.example.com` to TXTDirect's servers and when
+TXTDirect receives a request for `git.example.com`, it will look for a `TXT`
+record with the `_redirect.git.example.com` address. Then it will redirect the
+requests based on the `TXT` record's type and endpoint, which in case of this
+example, it redirects the requests without any additional check and logic to
+the endpoint specified in the `to=` field.
```
Comment (stp-ip) [inline content/docs/hosted/_index/basics.md]: Probably best not to use a git example, while explaining the host type. Feels like this would be better suited for a git type. So might be better to do a my-code.example.com and redirect to github or maybe do twitter.example.com and redirect to twitter?

File: content/docs/hosted/_index/basics.md [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+
+You can use all of TXTDirect's types and features on the hosted service too and
+the only thing that is different compared to running your instance is that
+you have to point each domain to `txtd.io` before writing its TXT records like
+the examples above.
+
+Take a look at the [Basic Configuration](/docs/config/basics/) page to learn
+more about TXTDirect's types and how they work. And [Specification](/docs/specification) page to read about the available types. And the
```
Comment (stp-ip) [inline content/docs/hosted/_index/basics.md]:
```suggestion
more about TXTDirect's types and how they work. The [Specification](/docs/specification) page details the available types. Additional
```

File: content/docs/hosted/_index/basics.md [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+[Record Examples](/docs/examples) provides a set of examples for each type and
```
Comment (stp-ip) [inline content/docs/hosted/_index/basics.md]:
```suggestion
[Record Examples](/docs/examples) give some ideas for each type and
```

File: content/docs/examples/combination/predefind/index.md [hunk 19]
```diff
@@ -0,0 +1,4 @@
++++
```
Comment (stp-ip) [inline content/docs/examples/combination/predefind/index.md]: Directory name should probably be `predefined` instead of `predefind`

File: content/docs/examples/_global/nav.md [hunk 20]
```diff
@@ -0,0 +1,20 @@
++++
```
Comment (stp-ip) [inline content/docs/examples/_global/nav.md:1]: Same here. Why the additional navigation?

File: content/docs/examples/combination/pathwithother/content.md [hunk 21]
```diff
@@ -0,0 +1,40 @@
++++
+fragment = "content"
+weight = 200
+
+title = "Path with Other"
```
Comment (stp-ip) [inline content/docs/examples/combination/pathwithother/content.md]:
```suggestion
title = "Path with other types"
```

File: content/docs/examples/combination/pathwithother/index.md [hunk 22]
```diff
@@ -0,0 +1,4 @@
++++
+title = "Path with Other"
```
Comment (stp-ip) [inline content/docs/examples/combination/pathwithother/index.md]:
```suggestion
title = "Path with other types"
```

File: content/docs/examples/combination/_index/content.md [hunk 23]
```diff
@@ -0,0 +1,12 @@
...
+title = "Combination"
+weight = 300
+
+[sidebar]
+  enable = true
++++
+
+TXTDirect types can work together to redirect or proxy the requests with more complex logic. For example, the `path` type can be combined with all of the other types so you can redirect the requests based on their path to different zones with different types.
+
+In this section, all of the possible scenarios that different types can be combined are listed on the left menu.
```
Comment (stp-ip) [inline content/docs/examples/combination/_index/content.md]: Assuming a specific layout within the content is problematic.

File: content/docs/examples/combination/pathwithother/content.md [hunk 24]
```diff
@@ -0,0 +1,40 @@
...
+_redirect.pkg.example.com           3600 IN TXT    "v=txtv0;type=path"
+_redirect.foo.pkg.example.com       3600 IN TXT    "v=txtv0;type=gometa;to=https://github.com/foo/foo"
+_redirect.bar.pkg.example.com       3600 IN TXT    "v=txtv0;type=gometa;to=https://github.com/foo/bar"
+_redirect.baz.pkg.example.com       3600 IN TXT    "v=txtv0;type=dockerv2;to=https://gcr.io/foo/baz"
+```
+
+Using these records we can setup the following redirects:  
+**pkg.example.com/foo -> https://github.com/foo/foo**  
+**pkg.example.com/bar -> https://github.com/foo/bar**  
+**pkg.example.com/baz -> https://gcr.io/foo/baz**
```
Comment (stp-ip) [inline content/docs/examples/combination/pathwithother/content.md]: These will have working links, which is problematic. They shouldn't be clickable as they are not useful links and will distract from the content.

File: content/docs/examples/combination/pathwithother/content.md [hunk 25]
```diff
@@ -0,0 +1,40 @@
...
+
+title = "Path with Other"
+
+[sidebar]
+  enable = true
++++
+
+The `path` type can also be used with other types like `dockerv2` or `host`.
+This feature enables you to redirect requests based on their path to different
+endpoints. For example, we use this feature to redirect create vanity URLs for
```
Comment (stp-ip) [inline content/docs/examples/combination/pathwithother/content.md]:
```suggestion
endpoints. For example, we use this feature to create vanity URLs for
```

File: content/docs/examples/combination/pathwithother/content.md [hunk 26]
```diff
@@ -0,0 +1,40 @@
...
+fragment = "content"
+weight = 200
+
+title = "Path with Other"
+
+[sidebar]
+  enable = true
++++
+
+The `path` type can also be used with other types like `dockerv2` or `host`.
```
Comment (stp-ip) [inline content/docs/examples/combination/pathwithother/content.md]:
```suggestion
The `path` type can also be used with other types like `gometa` or `host`.
```

File: content/docs/examples/combination/pathwithother/content.md [hunk 27]
```diff
@@ -0,0 +1,40 @@
...
+This feature enables you to redirect requests based on their path to different
+endpoints. For example, we use this feature to redirect create vanity URLs for
+our packages. Take a look at this example:
+
+```
+pkg.example.com                     3600 IN A      127.0.0.1
+_redirect.pkg.example.com           3600 IN TXT    "v=txtv0;type=path"
+_redirect.foo.pkg.example.com       3600 IN TXT    "v=txtv0;type=gometa;to=https://github.com/foo/foo"
+_redirect.bar.pkg.example.com       3600 IN TXT    "v=txtv0;type=gometa;to=https://github.com/foo/bar"
+_redirect.baz.pkg.example.com       3600 IN TXT    "v=txtv0;type=dockerv2;to=https://gcr.io/foo/baz"
```
Comment (stp-ip) [inline content/docs/examples/combination/pathwithother/content.md]:
```suggestion
_redirect.baz.pkg.example.com       3600 IN TXT    "v=txtv0;type=host;to=https://github.com/foo/baz/blob/master/README.md#deprecated"
```

Comment (stp-ip) [reply content/docs/examples/combination/pathwithother/content.md]: Using host as an additional example seems a bit simpler. Especially considering our current stability of dockerv2 :)

File: content/docs/examples/combination/pathwithother/content.md (hunk 24 above)
Comment (stp-ip) [inline content/docs/examples/combination/pathwithother/content.md]: Docker rewrite also has to be changed due to my suggestion.

File: content/docs/examples/combination/pathwithother/content.md [hunk 28]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+Using these records we can setup the following redirects:  
+**pkg.example.com/foo -> https://github.com/foo/foo**  
+**pkg.example.com/bar -> https://github.com/foo/bar**  
+**pkg.example.com/baz -> https://gcr.io/foo/baz**
+
+But these redirects are not just simple redirects because we've used `gometa` and
+`dockerv2` types in this example. To learn more about what each type can do
```
Comment (stp-ip) [inline content/docs/examples/combination/pathwithother/content.md]:
```suggestion
`host` types in this example. To learn more about what each type can do
```

File: content/docs/examples/combination/pathwithother/content.md [hunk 29]
```diff
@@ -0,0 +1,40 @@
...
+check the [Specification](/docs/specification) page.
```
Comment (stp-ip) [inline content/docs/examples/combination/pathwithother/content.md]:
```suggestion
check out the full [specification](/docs/specification).
```

File: content/docs/examples/combination/pathwithother/content.md [hunk 30]
```diff
@@ -0,0 +1,40 @@
...
+
+We can use the records from the example above in `go get` command and the
+Docker CLI. For example, we can use them like this:
```
Comment (stp-ip) [inline content/docs/examples/combination/pathwithother/content.md]: As mentioned about let's keep it simple with gometa and host. So probably an example with curl instead of docker. Or what do you think?

File: content/docs/examples/combination/predefind/content.md [hunk 31]
```diff
@@ -0,0 +1,41 @@
...
+title = "Predefined Regex"
+
+[sidebar]
+  enable = true
++++
+
+The `path` type has a feature called predefined regex that basically reads the
+`re=` field of the child zones and choses the record with most specific regex
+match. This feature enables you to have more advanced redirects and use
+different regexes in subzones to easily separate complex logics for `path`
```
Comment (stp-ip) [inline content/docs/examples/combination/predefind/content.md]:
```suggestion
different regexes in subzones to easily separate complex logic for `path`
```

File: content/docs/examples/combination/predefind/content.md [hunk 32]
```diff
@@ -0,0 +1,41 @@
...
+match. Take a look at the example below:
+
+```
+"_redirect.example.com.":     "v=txtv0;re=record;to=https://example.com;type=path",
+"_redirect.1.example.com.":   "v=txtv0;re=\\/test1;to=https://example.com/first/predefined{1};type=host",
+"_redirect.2.example.com.":   "v=txtv0;re=\\/test1\\/test2;to=https://example.com/second/predefined{1};type=host",
+```
+
+In the example above we have a `path` record which is `_redirect.example.com`
+and its `re=` field is set to `record`. When a request comes for `example.com`,
```
Comment (stp-ip) [inline content/docs/examples/combination/predefind/content.md]:
```suggestion
and its `re=` field is set to `record`. When a request comes in for `example.com`,
```

File: content/docs/examples/combination/predefind/content.md [hunk 33]
```diff
@@ -0,0 +1,41 @@
...
+TXTDirect looks for the numbered subdomains which start from
+`_redirect.1.example.com`. Then TXTDirect will match the request's path with the
+subzons `re=` field to find the most specific
+match. After finding the match, it will use that record to redirect the request.
+Which in this case if the incoming request's path is `/test1/test2`, TXTDirect
+will use the `_redirect.2.example.com` record and after parsing the placeholders,
+redirects the request to `https://example.com/second/predefined/test1/test2`.
```
Comment (stp-ip) [inline content/docs/examples/combination/predefind/content.md]:
```suggestion
`_redirect.1.example.com`. TXTDirect will then match the request's path with the
subzone's `re=` field to find a match. The order is defined by their numbered ordering.
After finding a match, it will use that record's `to=` field to redirect the request.

If the incoming request's path is `/test1/test2`, TXTDirect
will use the `_redirect.2.example.com` record and after parsing the placeholders,
redirects the request to `https://example.com/second/predefined/test1/test2`.
```

File: content/docs/examples/combination/predefind/content.md [hunk 34]
```diff
@@ -0,0 +1,41 @@
...
+
+For more information about the `path` records check the
+[Specification](/docs/specification/#path-type) page and for a list of
+placeholders and their description check the [Placeholders](/docs/placeholders)
+page.
```
Comment (stp-ip) [inline content/docs/examples/combination/predefind/content.md]: For more information about the `path` records check the
```suggestion
For more information about the `path` records check out our 
[specification](/docs/specification/#path-type) and the full list of our [placeholders](/docs/placeholders).
```

File: content/docs/examples/dockerv2/dockerv2.md [hunk 35]
```diff
@@ -9,30 +9,34 @@ title = "Dockerv2"
   enable = true
 +++
 
-__Currently Docker hub is not supported__
-__Only publicly available containers are currently supported__
+**Currently Docker hub is not supported**
```
Comment (stp-ip) [inline content/docs/examples/dockerv2/dockerv2.md]: Stability note as well similar to "proxy"? Aka experimental?

**Review by stp-ip — COMMENTED** [review]
Wow that was a review. Also seems like I had an older review done, but it was never sent. Sorry for that.

Anyway full review. Overall \o/. Damn that makes it so much easier for me to figure out configs from now on :)

File: content/docs/config/fallback/content.md (hunk 12 above)
Comment (erbesharat) [reply content/docs/config/fallback/content.md:16]: In case of path chaining, if the second record triggers the fallback, we shouldn't use the parent record's to= field as the first option like the current record's fallback flow because the parent record's to= field is pointed to the second record which triggered fallback.

File: content/docs/examples/_global/nav.md (hunk 20 above)
Comment (erbesharat) [reply content/docs/examples/_global/nav.md:1]: It's because of my lack of knowledge about Syna's structure :smile:. Mahdi once told me about this but it seems like I forgot to fix it.

File: content/docs/placeholders/placeholders.md (hunk 7 above)
Comment (erbesharat) [reply content/docs/placeholders/placeholders.md:1]: Currently, "static" placeholders would overwrite the regex placeholders. I prefer the prefixes to be short, so something like `{_my-regex}` or `{$my-regex}` seems better to me.

File: content/docs/config/basics/types.md (hunk 3 above)
Comment (erbesharat) [reply content/docs/config/basics/types.md:1]: I don't like the table design either but couldn't think of a better way to provide a good overview for types and placeholders. Would be happy to change it if you have other ideas but if not, I guess it would be good to open an issue as you said and get back to it later :)

File: content/docs/config/_index/content.md (hunk 1 above)
Comment (erbesharat) [reply content/docs/config/_index/content.md]: Would it sound better if we only mentioned the documentation navigation menu and not its position on the page? Can't really think of a better way to point to the menu.

Comment (stp-ip) [reply content/docs/config/_index/content.md]: That would be a start. Or we could link to the docs in general as the overview page should lead them there as well.
Something along the lines of.
These features are documented in more detail in separate sections within our [docs].

File: content/docs/config/basics/content.md (hunk 5 above)
Comment (stp-ip) [reply content/docs/config/basics/content.md]: ?

File: content/docs/config/basics/content.md (hunk 4 above)
Comment (stp-ip) [reply content/docs/config/basics/content.md]: ?

File: content/docs/examples/combination/_global/nav.md [hunk 36]
```diff
@@ -0,0 +1,20 @@
++++
```
Comment (stp-ip) [inline content/docs/examples/combination/_global/nav.md]: Why the additional nav here as well?

File: content/docs/examples/combination/pathwithother/content.md (hunk 24 above)
Comment (stp-ip) [reply content/docs/examples/combination/pathwithother/content.md]: ?

Comment (stp-ip) [reply content/docs/examples/combination/pathwithother/content.md]: ?

File: content/docs/examples/combination/pathwithother/content.md (hunk 30 above)
Comment (stp-ip) [reply content/docs/examples/combination/pathwithother/content.md]: ?

File: content/docs/hosted/_index/basics.md (hunk 16 above)
Comment (stp-ip) [reply content/docs/hosted/_index/basics.md]: ?

**Comment by stp-ip** [comment]
Damn gh review doesn't show the parent comment in a new review. All the "?" were responses to unresolved comments.

File: content/docs/hosted/_index/content.md (hunk 14 above)
Comment (erbesharat) [reply content/docs/hosted/_index/content.md:18]: I actually tried to write something similar to yc guide but for a self-hosted DNS server. I guess a good thing would be providing separate guides for each DNS provider to make the hosted version as easy as possible to understand and use. It can either be a set of screenshots and descriptions like yc or videos.

File: content/docs/config/_index/content.md [hunk 37]
```diff
@@ -35,7 +35,8 @@ example.com:8080 {
 }
 ```
 
-These features are documented in separate sections that you can find using the menu on left side of the page.
+These features are documented in more detail in separate sections within our
+[documentation](/docs) under the configuration category.
```
Comment (stp-ip) [inline content/docs/config/_index/content.md]: Why not link to the specific section?

File: content/docs/specification/content.md [hunk 38]
```diff
@@ -191,7 +195,7 @@ If the request's path is neither of those cases, it would then use the last wild
 
 - Key: **from**
 - Permitted values: "simplified regex"
-- Example: "from=/$2/$1/$3"
+- Example: "from=/$2/$1/\$3"
```
Comment (stp-ip) [inline content/docs/specification/content.md]: Not intentional I assume :)

**Review by stp-ip — CHANGES_REQUESTED** [review]
2 small nits. Besides this ready to merge.

**Review by stp-ip — APPROVED** [review]
